configtx/example_test.go

```diff
@@ -561,6 +561,160 @@ func ExampleNewSystemChannelGenesisBlock() {
 	}
 }
 
+func ExampleNewApplicationChannelGenesisBlock() {
+	channel := configtx.Channel{
+		Orderer: configtx.Orderer{
+			Policies: map[string]configtx.Policy{
+				configtx.ReadersPolicyKey: {
+					Type: configtx.ImplicitMetaPolicyType,
+					Rule: "ANY Readers",
+				},
+				configtx.WritersPolicyKey: {
+					Type: configtx.ImplicitMetaPolicyType,
+					Rule: "ANY Writers",
+				},
+				configtx.AdminsPolicyKey: {
+					Type: configtx.ImplicitMetaPolicyType,
+					Rule: "MAJORITY Admins",
+				},
+				configtx.BlockValidationPolicyKey: {
+					Type: configtx.ImplicitMetaPolicyType,
+					Rule: "ANY Writers",
+				},
+			},
+			OrdererType: orderer.ConsensusTypeSolo,
```

Comment:
As this API will be used almost explicitly for the channel participation API/no system channel case which only supports etcdraft configurations, could you maybe update this example to use `ConsensusTypeEtcdRaft`?